config/global.ini.php

```diff
@@ -744,6 +744,12 @@
 
 [Tracker]
 
+; When enabled and a userId is set, then the visitorId will be automatically set based on the userId. This allows to
+; identify the same user as the same visitor across devices. 
+; Disabling this feature can be useful for example when using the third party cookie, and thus all Matomo sites use
+; the same "global" visitorId for the same device, and some Matomo sites set a userid.
```

Comment:
would change 'and thus all Matomo sites use the same "global" visitorId for the same device, and some Matomo sites set a userid.' to 'where all Matomo tracked sites use the same "global" visitorId for a device and you want to see when the same user switches between devices.'

Reply:
Updated 👍